spin/cmds/meson.py

```diff
@@ -601,3 +601,29 @@ def docs(ctx, sphinx_target, clean, first_build, jobs, sphinx_gallery_plot):
         f"$ export PYTHONPATH={os.environ['PYTHONPATH']}", bold=True, fg="bright_blue"
     )
     _run(["make", "-C", "doc", sphinx_target], replace=True)
+
+
+@click.command()
```

Comment:
Place this command in a new file, called `pip.py`

Reply:
Done